src/pudl/output/ferc1.py

```diff
@@ -1259,37 +1263,43 @@ def exploded_table_asset_factory(
 ) -> AssetsDefinition:
     """Create an exploded table based on a set of related input tables."""
     ins: Mapping[str, AssetIn] = {
-        "metadata_xbrl_ferc1": AssetIn("metadata_xbrl_ferc1"),
-        "calculation_components_xbrl_ferc1": AssetIn(
-            "calculation_components_xbrl_ferc1"
+        "_core_ferc1_xbrl__metadata": AssetIn("_core_ferc1_xbrl__metadata"),
+        "_core_xbrl_ferc1__calculation_components": AssetIn(
+            "_core_xbrl_ferc1__calculation_components"
         ),
         "_out_ferc1__explosion_tags": AssetIn("_out_ferc1__explosion_tags"),
     }
     ins |= {table_name: AssetIn(table_name) for table_name in table_names}
 
-    @asset(name=f"exploded_{root_table}", ins=ins, io_manager_key=io_manager_key)
+    @asset(
+        name=f"_out_ferc1__exploded_{root_table.replace('_ferc1', '')}",
```

Comment:
I think the root_table here is the full core table name like `core_ferc1__yearly_income_statements_sched114`. So i think we'll need to remove the long  `core_ferc1__yearly_` prefix and the `_sched###` suffix

Reply:
The names look correct on my dagster UI. I think the root_table uses the raw/old names? Maybe these should also be renamed:

https://github.com/catalyst-cooperative/pudl/blob/0b98ecb8382822aa871a7a3537d0ebdbafeed173/src/pudl/output/ferc1.py#L1319-L1327

Reply:
oooooooh i see yes these `root_table`s should definitely rename these guys to the new names!

Reply:
Ok! I've run into some issues when renaming inputs and outputs of the explosion assets. Any idea which metadata spreadsheet need to be updated when changing the `root_table` names?

Reply:
from what i can tell the `root_table` is used entirely for logging purposes... can you share the errors?

Reply:
Oh amazing. I actually did not run into any issues. Just pushed the changes.